2021-01-24 17:13:50,657 INFO    MainThread:7296 [wandb_setup.py:_flush():70] setting env: {}
2021-01-24 17:13:50,657 INFO    MainThread:7296 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-01-24 17:13:50,657 INFO    MainThread:7296 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-24 17:13:50,657 INFO    MainThread:7296 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-24 17:13:50,658 INFO    MainThread:7296 [wandb_init.py:_log_setup():318] Logging user logs to /home/iiitd/Desktop/Ahmad/cnn/wandb/run-20210124_171350-xwks4za6/logs/debug.log
2021-01-24 17:13:50,658 INFO    MainThread:7296 [wandb_init.py:_log_setup():319] Logging internal logs to /home/iiitd/Desktop/Ahmad/cnn/wandb/run-20210124_171350-xwks4za6/logs/debug-internal.log
2021-01-24 17:13:53,266 INFO    MainThread:7296 [wandb_run.py:_console_start():1308] atexit reg
2021-01-24 17:13:53,266 INFO    MainThread:7296 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-01-24 17:13:53,267 INFO    MainThread:7296 [wandb_run.py:_redirect():1178] Redirecting console.
2021-01-24 17:13:53,267 INFO    MainThread:7296 [redirect.py:install():205] install start
2021-01-24 17:13:53,270 INFO    MainThread:7296 [redirect.py:install():220] install stop
2021-01-24 17:13:53,270 INFO    MainThread:7296 [redirect.py:install():205] install start
2021-01-24 17:13:53,270 INFO    MainThread:7296 [redirect.py:install():220] install stop
2021-01-24 17:13:53,270 INFO    MainThread:7296 [wandb_run.py:_redirect():1222] Redirects installed.
2021-01-24 17:13:53,271 INFO    MainThread:7296 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-01-24 17:13:53,271 INFO    MainThread:7296 [wandb_run.py:_restore():1250] restore
2021-01-24 17:13:53,271 INFO    MainThread:7296 [redirect.py:uninstall():224] uninstall start
2021-01-24 17:13:53,272 INFO    MainThread:7296 [redirect.py:_stop():279] _stop: stdout
2021-01-24 17:13:53,272 INFO    MainThread:7296 [redirect.py:_stop():285] _stop closed: stdout
2021-01-24 17:13:53,272 INFO    stdout    :7296 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-01-24 17:13:53,272 INFO    stdout    :7296 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-01-24 17:13:53,272 INFO    MainThread:7296 [redirect.py:_stop():291] _stop joined: stdout
2021-01-24 17:13:53,272 INFO    MainThread:7296 [redirect.py:_stop():293] _stop rd closed: stdout
2021-01-24 17:13:53,272 INFO    MainThread:7296 [redirect.py:uninstall():228] uninstall done
2021-01-24 17:13:53,272 INFO    MainThread:7296 [redirect.py:uninstall():224] uninstall start
2021-01-24 17:13:53,272 INFO    MainThread:7296 [redirect.py:_stop():279] _stop: stderr
2021-01-24 17:13:53,272 INFO    MainThread:7296 [redirect.py:_stop():285] _stop closed: stderr
2021-01-24 17:13:53,272 INFO    stderr    :7296 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-01-24 17:13:53,272 INFO    stderr    :7296 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-01-24 17:13:53,273 INFO    MainThread:7296 [redirect.py:_stop():291] _stop joined: stderr
2021-01-24 17:13:53,273 INFO    MainThread:7296 [redirect.py:_stop():293] _stop rd closed: stderr
2021-01-24 17:13:53,273 INFO    MainThread:7296 [redirect.py:uninstall():228] uninstall done
2021-01-24 17:13:53,939 INFO    MainThread:7296 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "d0dfb82bdf624004a3ae09f693dddfd1"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 511
      total_bytes: 511
    }
  }
}

2021-01-24 17:13:55,943 INFO    MainThread:7296 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "2424b6b4154f4351ba41811b44fca7ba"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 6107
      total_bytes: 6107
    }
  }
}

2021-01-24 17:13:57,244 INFO    MainThread:7296 [wandb_run.py:_show_files():1614] logging synced files
2021-01-24 17:13:57,245 INFO    MainThread:7296 [internal.py:handle_exit():139] Internal process exited
